CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,5 @@
+<!-- alex disable-->
```

Comment:
Thought it was safe to ignore this file

Reply:
I should clarify: the content causes alerts (4 I think) which are obviously not needed. We should be able to control this file ourselves.

Reply:
I think this is going to be the only file that would ever be ignored, which is why I chose this option.

It means a lot to have had your eyes on this @ricknahlonii. I'm greatful that you took the time to do this in the React Native repo. It's something I use almost every day at work - thank you 🙌